src/editors/containers/TextEditor/components/SelectImageModal/hooks.js

```diff
@@ -36,9 +35,7 @@ export const displayList = ({ sortBy, searchString, images }) => (
     imageList: Object.values(images),
   }).sort(sortFunctions[sortBy in sortKeys ? sortKeys[sortBy] : sortKeys.dateNewest]));
 
-export const imgListHooks = ({ searchSortProps, setSelection }) => {
-  const dispatch = useDispatch();
-  const [images, setImages] = module.state.images({});
+export const imgListHooks = ({ searchSortProps, setSelection, images }) => {
```

Comment:
When I try running this in www gallery, I get the error `Cannot convert undefined or null to object`. I'm guessing this has to do with images being null by default.

Reply:
yes, and gallery doesn't use any api calls so it won't be able to pull anything. Should I make it so that it defaults to an empty object?

Reply:
Yes, I think that should work.